client-side/src/Components/adminComponent/Settings/group.js

```diff
@@ -1,36 +1,47 @@
-
 import React, { useState, useEffect } from "react";
 import Navbar from "../../../Shared/Navbar/navbar";
 import Sidebar from "../../../Shared/Sidebar/adminSidebar";
 import "./group.scss";
 
-
 export const Group = () => {
   const [groups, setGroups] = useState([]);
   const [subgroups, setSubgroups] = useState([]);
-  const counter = () =>{
-    let counter1 = 0;
-    counter1++;
-    return counter1;
-  }
+  const [showMore, setShowMore] = useState(false);
+  const [showMore1, setShowMore1] = useState(false);
```

Comment:
change variable names to showMoreGroup and showMoreSubgroup

Reply:
Hi I have done it as per your request, please accept the changes

Reply:
Corrected the issue, Thanks